src/components/domains/conversation/views/PreviewView.js

```diff
@@ -5,23 +5,30 @@ import Activity from "src/components/domains/activity/Activity";
 import NameAndIcon from "src/components/domains/member/NameAndIcon";
 
 export default function PreviewView(props) {
-  let { activity, handlers, onExpand, term } = props;
+  let { activity, conversation, handlers, onExpand, term } = props;
   let { onClickMember } = handlers;
 
-  const conversation = activity.conversation;
   const descendants = conversation.descendants;
+  const starter = descendants[0];
```

Comment:
Is there a way to guarantee at the DB level this can never be undefined? Meaning conversations must have at least one descendant

Reply:
At the graphql level it is:

`descendants: [Activity!]! @relationship(type: "INCLUDES", direction: OUT)`

If a query asks for descendants and there aren't any, it will error out. At the DB level it's a good question, it would probably be a separate kind of check. I'm not sure that relationship-level asserts are supported by our graph db.